packages/daemon/src/service/pipeline.ts

```diff
@@ -317,6 +317,66 @@ export class PipelineService {
     return path.join(CoreConstants.PIPCOOK_RUN, id, 'output.tar.gz');
   }
 
+  // private _generateWASMOutput(dist: string, opts: GenerateOptions, fileQueue: Array<Promise<void | string>>): void {
```

Comment:
@WenheLI @FeelyChau Boa is working with `worker_threads` at #602, this function could be reimplemented with `worker_threads` to avoid blocking the server.